modules/claims_api/app/controllers/claims_api/v2/blueprints/power_of_attorney_blueprint.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  module V2
+    module Blueprints
+      class PowerOfAttorneyBlueprint < Blueprinter::Base
+        identifier :code
+        field :name
+        field :type
+        field :phone do |entity, _options|
```

Comment:
is `_options` needed, or is this just to make one aware that it exists if needed later and the prepended `_` is the way to make `rubocop` ok with an used argument?

Reply:
the latter